samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -814,6 +835,9 @@ int link_shell(const struct shell *shell, size_t argc, char **argv)
 			break;
 		case 'd':
 			link_cmd_args.common_option = LINK_COMMON_DISABLE;
+			if (link_cmd_args.command == LINK_CMD_REDMOB) {
+				link_cmd_args.redmob_mode = LTE_LC_REDUCED_MOBILITY_DISABLED;
+			}
```

Comment:
I would rather not add special handling for reduced mobility here, but instead check the `common_option` for `LINK_COMMON_DISABLE` below where `LINK_CMD_REDMOB` is handled.

Reply:
Should be fixed now.